src/Peachpie.Library.Graphics/PhpGd2.cs

```diff
@@ -426,7 +426,18 @@ public static PhpResource imagecreatefromstring(byte[] image)
 
             try
             {
-                return new PhpGdImageResource(Image.Load(image, out var format), format);
+                var img = Image.Load<Rgba32>(image, out var format);
+
+                // PHP will only preserve the first frame of a multi-frame image.
+                if (img.Frames.Count > 0)
+                {
+                    for (int i = 1; i < img.Frames.Count; i++)
+                    {
+                        img.Frames.RemoveFrame(i);
+                    }
+                }
```

Comment:
shuldn't it be more like:
```
while (img.Frames.Count > 1)
{
  img.Frames.RemoveFrame(img.Frames.Count - 1);
}
```
otherwise it would remove just the every second frame, and every removed frame would cause an unnecessary copying (since removing from 1 to count)

Reply:
Whoops! Yes, should be `>1’ that should teach me to write code after midnight!

Reply:
Haha and there shouldn't be `i++` :)) ANYWAYS thanks for the pull request!